.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ruby.rubocop.useBundler": true
+}
```

Comment:
this file should not be checked into the repo, this can be done by including `.vscode/* in the `.gitignore` file